.github/dependabot.yml

```diff
@@ -15,3 +15,12 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      dev-deps:
+        dependency-type: "development"
+      prod-deps:
+        dependency-type: "production"
```

Comment:
```suggestion
```

I think it'd be better to consider each update independently (i.e. remove the groups). That would look like this:

- https://github.com/victorlin/augur/pull/11
- https://github.com/victorlin/augur/pull/10
- https://github.com/victorlin/augur/pull/9
- https://github.com/victorlin/augur/pull/8
- https://github.com/victorlin/augur/pull/7